unit_test/plugin_examples/supported_version_validator/plugin.spec_bad_empty.yaml

```diff
@@ -0,0 +1,259 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: active_directory_ldap
+title: Active Directory LDAP
+description: This plugin utilizes Microsoft\'s Active Directory service to create and manage domains, users, and objects within a network
+version: 3.2.7
+vendor: rapid7
+support: ass
```

Comment:
lol

Reply:
How did this pass the profanity validator lol